### DISCUSSION TITLE
Copilot for Webstorm never showing more than one solution

### DISCUSSION THREAD

**Discussion by OrbiterToad** [community/community#8951] [Copilot Conversations]
Hey Hey,

Thanks for the copilot, helps a lot!

But in Webstorm Copilot only ever shows one solution... Not sure if this is yet to be implemented like in VScode (Next Suggestion)

Is this a bug or intended?

Best Regards

**Comment by gsusI** [comment]
Any updates on this?

**Reply by jansorg** [reply]
Could you share a code snippet whee this happens, please? Support for multi-line snippets differs by the file type, for example.

**Comment by OrbiterToad** [comment]
this still happens on smaller code completions (only one answer)
but on larger ones creating multiple lines of code there are often more than one solutions
so, i think this can be closed
